parser/parser.go

```diff
@@ -98,10 +98,18 @@ func (p *LogParser) ParseLine(line string) models.LogEntry {
 	finalParts := make([]string, 0)
 
 	var fieldPosAdjustment int
+	var isTimestampAlreadyFound = false
 	// Validate each part
 	for i, part := range parts {
 		fieldType := determineField(part)
 		expectedType := expectedFieldType(i + fieldPosAdjustment)
+		if fieldType == dateTimeField {
```

Comment:
Move this block to original line number 116. You will also have to do fieldPosAdjustment++ before continue.